config/setupMPI.cmake

```diff
@@ -245,32 +249,41 @@ macro( setupOpenMPI )
     set( MPIEXEC_PREFLAGS "$ENV{MPIEXEC_PREFLAGS}" )
   endif()
 
-  # Notes:
-  # - For PERFBENCH that use Quo, we need '--map-by socket:SPAN' instead of '-bind-to none'.  The
-  #   'bind-to none' is required to pack a node.
-  # - Adding '--debug-daemons' is often requested by the OpenMPI dev team in conjunction with
-  #   'export OMPI_MCA_btl_base_verbose=100' to obtain debug traces from openmpi.
-  set(MPIEXEC_PREFLAGS_PERFBENCH "${MPIEXEC_PREFLAGS} --map-by socket:SPAN")
-  if( NOT MPIEXEC_PREFLAGS MATCHES " -bind-to none")
-    string(APPEND MPIEXEC_PREFLAGS " -bind-to none")
-  endif()
-  # Setup for OMP plus MPI
-  if( NOT APPLE AND NOT MPIEXEC_OMP_PREFLAGS MATCHES "--map-by ppr")
-    # -bind-to fails on OSX, See #691
-    set(MPIEXEC_OMP_PREFLAGS
-      "${MPIEXEC_PREFLAGS} --map-by ppr:${MPI_CORES_PER_CPU}:socket --report-bindings" )
-  endif()
+  if("${MPIEXEC_EXECUTABLE}" MATCHES "srun")
+    set(preflags " --overlap") # -N 1 --cpu_bind=verbose,cores
```

Comment:
`--overlap` seems to be the better choice for our testing system.  `--exclusive` alters what the machine looks like to the code by hiding resources not explicitly requested. E.g.: `srun -n 4` will look like the hardware only has 4 cores.

Reply:
Will this cause any issues with the quo tests that query and use all available resources on the node? Maybe they were already running into other tests under the old run command.

Reply:
The new `srun` model breaks Quo in the way we want to use it.  We will need to adapt to the new system.  I think the best way forward is `--overlap` since this option allows a job to see the whole node.   `--exclusive` has a new meaning that restricts the application to only the number of MPI ranks specified by `-n`.  That is something like `srun -n 1 --exclusive tstFoo` can only use one core -- even if the remainder of the node is empty.

If we have tests that are using too many ranks, we need to change how those tests are registered with `ctest`.

Reply:
OK, sounds good. I guess we'll see if the runtime on those quo tests changes drastically.